azurerm/internal/services/keyvault/key_vault_key_encrypt_resource.go

```diff
@@ -0,0 +1,94 @@
+package keyvault
```

Comment:
If the intention is to encrypt secrets, might i suggest that the resource instead be called:
`azurerm_key_vault_secret_encrypt`

Reply:
becasue key vault key and key vault secret are different resource, and the rest api is using key vault key to encrypt and decrypt data. So personally I think `azurerm_key_vault_secret_encrypt` maybe a little misleading?